python/lsst/ip/isr/crosstalk.py

```diff
@@ -206,6 +211,7 @@ def fromDetector(self, detector, coeffVector=None, coeffSqrVector=None):
         self.coeffValid = np.ones(self.crosstalkShape, dtype=bool)
         self.coeffErrSqr = np.zeros(self.crosstalkShape)
         self.ampGainRatios = np.zeros(self.crosstalkShape)
+        self.fitGains = np.zeros(self.nAmp)
```

Comment:
I don't know if this was an intentional choice, but should this be `np.ones` here instead?  The default casts this to zeros as well, but I can't think of any gain operations that can be performed where zero yields a usable answer.

Reply:
This was intentional in that I wanted zero to mean "this calib did not have a fitGain so don't try to use it".  (And it is not in use anywhere).  But in use I think that would warn and set to 1.0; I could set here to 1.0 and not worry about it downstream if you prefer.